src/main/java/org/gridsuite/study/server/service/LoadflowService.java

```diff
@@ -97,8 +99,8 @@ public void runLoadFlow(UUID studyUuid, UUID nodeUuid, LoadFlowParametersInfos l
             result = resp.getBody();
             updateLoadFlowResultAndStatus(nodeUuid, result, computeLoadFlowStatus(result), false);
         } catch (Exception e) {
-            updateLoadFlowStatus(nodeUuid, LoadFlowStatus.NOT_DONE);
-            throw e;
+            updateLoadFlowResultAndStatus(nodeUuid, null, LoadFlowStatus.DIVERGED, false);
```

Comment:
In this catch, there can be technical errors. I'm not sure about using 'DIVERGED' in all possible cases

Reply:
Right !
I added a new loadflow status : FAILED